src/Sulu/Bundle/PageBundle/Resources/translations/admin.en.json

```diff
@@ -57,5 +57,8 @@
     "sulu_page.segment_label": "Assign segment",
     "sulu_page.segment_description": "The block will be automatically filtered on the website by the chosen segment.",
     "sulu_page.target_groups_label": "Assign target groups",
-    "sulu_page.target_groups_description": "The block will only be visible for the selected target groups."
+    "sulu_page.target_groups_description": "The block will only be visible for the selected target groups.",
+    "sulu_page.add_schedule": "Add schedule",
+    "sulu_page.schedules_label": "Assign time schedule",
+    "sulu_page.schedules_description": "The block will only be visible if at least one condition is valid."
```

Comment:
```suggestion
    "sulu_page.schedules_description": "The block will only be visible if at least one condition is fulfilled."
```

Reply:
This translation is from Bernd, and he didn't like fulfilled.